addons/nuxeo-csv/nuxeo-csv-core/src/test/java/org/nuxeo/ecm/csv/core/operation/TestImportOperation.java

```diff
@@ -142,4 +143,53 @@ public void testImportOperation() throws OperationException, InterruptedExceptio
         assertEquals(336, result.getSuccessLineCount());
         assertEquals(336, result.getTotalLineCount());
     }
+    
+    @Test
+    public void TestImportIgnoreSurroundingSpaces() throws OperationException, InterruptedException {
+    	Map<String, Object> params = new HashMap<>();
+        params.put("path", testFolder.getPathAsString());
+
+        chain = new OperationChain("test-chain");
+        chain.add(CSVImportOperation.ID).from(params);
+
+        OperationContext ctx = new OperationContext(session);
```

Comment:
Please always use this pattern:
```
try (OperationContext ctx = new OperationContext(session)) {
   // do something with ctx
}
```
to make sure to properly close the context that implements `AutoCloseable`.
Else, the IDE should warn about "Potential resource leak: 'ctx' may not be closed at this location".